### DISCUSSION TITLE
VLAN MacOS

### DISCUSSION THREAD

**Discussion by lowrisk75** [utmapp/UTM#4620] [UTM Help (macOS)]
I'm running a VM for Home Assistant on a mac, at the moment I'm using a USB-C to ethernet which is disabled on MacOS and only used by the VM.

However I think it would be better if I sent a trunk to MacOS with a native VLAN for MacOS and two tag VLAN I could use for the VM
is it possible to connect to two VLAN this way on UTM? or how should I proceed?

**Comment by tkkost** [comment]
Did you find a solution?
